modules/images-create-metadata.adoc

```diff
@@ -53,11 +49,7 @@ LABEL io.openshift.tags   mongodb,mongodb24,nosql
 ====
 
 |`*io.openshift.wants*`
```

Comment:
According to the current guidelines, these elements should be styled in just backticks: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Reply:
@kalexand-rh, we seem to have quite a few of these in 22 files throughout the docs; I'll create a PR for that.